R-packages/evalcast/R/plot_canonical.R

```diff
@@ -0,0 +1,93 @@
+#' A canonical function for creating plots within evalcast
+#'
+#' @param df dataframe containing the data to plot
+#' @param x Name of the variable in `df` to be used for the x-axis
+#' @param y name of the variable in `df` to be used for the y-axis
+#' @param aggr function to use to aggregate data over `group_vars`
+#' @param dots when set to TRUE, a dot layer is included in the plot
+#' @param lines when set to TRUE, a line layer is included in the plot
+#' @param group_vars The set of variable names in `df` which to aggregate over.
+#'   "forecaster" by default.
+#' @param facet_rows A variable name to facet data over. Creates a
+#'   separate row of plots for each value of specified variable. Can be used
+#'   with `facet_cols` to create a grid of plots. 
+#' @param facet_cols Same as `facet_rows`, but with columns.
+#' @param base_forecaster If set, scales they y-value for all forecasters by
+#'   the corresponding values of the base_forecaster
+#' @param scale_before_aggregation If TRUE, scales results by the
```

Comment:
Let's mention that this is FALSE by default.

Reply:
Done